packages/central-server/src/tests/hooks/questionHooks.test.js

```diff
@@ -299,8 +299,8 @@ describe('Question hooks', () => {
         expect(beforeData).to.deep.equal(afterData);
       });
 
-      it("Should add household_head to entity's attributes when there are existing attributes", async () => {
-        const TEST_HOUSEHOLD_HEAD = 'Will Smith';
+      it("Should add custom attribute to entity's attributes when there are existing attributes", async () => {
```

Comment:
Scope for a third test here to make sure it successfully overwrites any existing value with the same key. Not a requirement, but if you have time :-)